src/SFA.DAS.AANHub.Data/AANDataContext.cs

```diff
@@ -22,6 +24,8 @@ public class AanDataContext : DbContext, IAanDataContext
     public DbSet<Notification> Notifications => Set<Notification>();
     public DbSet<NotificationTemplate> NotificationTemplates => Set<NotificationTemplate>();
 
+    public DbSet<Preference> Preference => Set<Preference>();
```

Comment:
Are we reading Preferences directly anywhere?

Reply:
Removed